plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -422,6 +487,11 @@ fu_thunderbolt_device_setup_controller(FuDevice *device, GError **error)
 		fu_device_add_internal_flag(device, FU_DEVICE_INTERNAL_FLAG_REPLUG_MATCH_GUID);
 	}
 
+	if (self->device_type == FU_THUNDERBOLT_DEVICE_TYPE_HOST_CONTROLLER) {
+		if(fu_thunderbolt_device_set_port_offline(FU_UDEV_DEVICE(device)))
```

Comment:
whitespace is still wrong

Reply:
fixed now https://github.com/fwupd/fwupd/pull/4044/commits/a9e404abce61dd403ee8ce7290b22462b0dcc720 with clang thank you!